resolver_conn_wrapper.go

```diff
@@ -43,7 +43,7 @@ type ccResolverWrapper struct {
 func split2(s, sep string) (string, string, bool) {
 	spl := strings.SplitN(s, sep, 2)
 	if len(spl) < 2 {
-		return "", "", false
+		return "", s, false
```

Comment:
We don't need `s` either, so we return `""` to save (a very small amount of) memory... See [the comment here](https://github.com/grpc/grpc-go/pull/1611#discussion_r146417755).

How about changing the comment to `returns ("", "", false)`?